packages/embark/src/lib/modules/blockchain_connector/provider.js

```diff
@@ -14,7 +14,12 @@ class Provider {
     this.web3Endpoint = options.web3Endpoint;
     this.logger = options.logger;
     this.isDev = options.isDev;
+    this.events = options.events;
     this.nonceCache = {};
+
+    this.events.setCommandHandler("blockchain:provider:contract:accounts:get", cb => {
+      cb(this.accounts);
+    });
```

Comment:
That's not really necessary. Blockchain connector already has a getAccounts event. `blockchain:getAccounts`